emails/tests/views_tests.py

```diff
@@ -711,6 +711,82 @@ def test_plain_text(self) -> None:
         assert self.ra.last_used_at
         assert (datetime.now(tz=timezone.utc) - self.ra.last_used_at).seconds < 2.0
 
+    def test_from_with_unquoted_commas_is_parsed(self) -> None:
+        """
+        A From: header with commas in an unquoted display is forwarded.
+
+        AWS parses these headers as a single email, Python as a list of emails.
+        One of the root causes of MPP-3407.
+        """
+        email_text = EMAIL_INCOMING["emperor_norton"]
+        test_sns_notification = create_notification_from_email(email_text)
+
+        # Python parses differently than AWS
```

Comment:
question (non-blocking): do we need these `Python parses differently than AWS` blocks of assertions? It looks like they are effectively asserting that `create_notification_from_email` and `json.loads` work like we expect?

Reply:
Hmm, now that I think about it, this implies these _should_ be SNS JSON fixtures, since the trip from a raw email doesn't work.